app/models/event_schedule.rb

```diff
@@ -37,6 +39,16 @@ def replacement?
 
   private
 
+  def start_after_end_hour
+    return unless errors.blank? && event.program.conference.try(:end_hour)
```

Comment:
@Ana06 
But we are not checking the presence start_time here.
`validate :start_after_end_hour if event.try(:program).try(:conference).try(:end_hour)`
We can do something like this
`validate :start_after_end_hour if (event.try(:program).try(:conference).try(:end_hour) && start_time)`
But then it will become too long for single line if. Therefore I was thinking to do this in validation body.